### ISSUE TITLE
FLOPs for DIT

### ISSUE THREAD

**Issue by Ethereal0725** [OpenSparseLLMs/Skip-DiT#1]
Hello,
I'm seeking guidance on how to measure the FLOPS performance metric specifically for DIT. Could you provide a code snippet or method to accurately calculate this metric? Your assistance is greatly appreciated.

**Comment by guanjiechen118** [comment]
Here is the pseudocode in python:

``` python
from thop import profile 
if __name__ == '__main__':
    device = "cuda" if torch.cuda.is_available() else "cpu"
    network = YOUR_DIT_MODEL
    low_cpu_mem_usage=False, device_map=None, video_length=16).to(device,  dtype=torch.float16)
    # simulate the inputs of the model
    h = torch.randn(...).to(device, dtype=torch.float16)
    e =  torch.randn(...).to(device,  dtype=torch.float16)
    t = 1000 # or other timestep you want to measure
    t = torch.tensor([t, t]).to(device,  dtype=torch.float16)
    flops, params = profile(network, inputs= (h,t,e))
    print('FLOPs = ' + str(flops/1000**3) + 'G')
    print('Params = ' + str(params/1000**2) + 'M')
```

Please replace the network and simulated inputs with yours
